### PR TITLE
[xc-admin] Crank : Improve error logging

### PR DESCRIPTION
**Pull request by guibescos**

### CODE REVIEW

File: governance/xc_admin/packages/crank_executor/src/index.ts [hunk 1]
```diff
@@ -213,5 +213,9 @@ async function run() {
 }
 
 (async () => {
-  await run();
+  try {
+    await run();
+  } catch (err) {
+    console.error(err);
+  }
```
Comment (jayantk) [inline governance/xc_admin/packages/crank_executor/src/index.ts:220]: this no longer throws on errors -- is that intentional?

**Review by jayantk — APPROVED** [review]

File: governance/xc_admin/packages/crank_executor/src/index.ts (hunk 1 above)
Comment (jayantk) [reply governance/xc_admin/packages/crank_executor/src/index.ts:220]: i mean it's not just improving logging, it's also suppressing errors

Comment (guibescos) [reply governance/xc_admin/packages/crank_executor/src/index.ts:220]: I'll make it throw...

Comment (guibescos) [reply governance/xc_admin/packages/crank_executor/src/index.ts:220]: Without this trick the error wasn't getting printed

**Pull request merged**